packages/swayze/lib/src/core/intents/selection_intents.dart

```diff
@@ -88,3 +88,40 @@ class HeaderSelectionUpdateIntent extends SwayzeIntent {
     required this.axis,
   });
 }
+
+// TODO: [victor] doc
+class HeaderDragStartIntent extends SwayzeIntent {
```

Comment:
Currently, it is in the `selection_intents` file, we can move it to a better place.